test/integration/test_backend.py

```diff
@@ -131,6 +132,7 @@ def test_backend_properties(self):
             if backend.simulator:
                 raise SkipTest("Skip since simulator does not have properties.")
             self.assertIsNotNone(backend.properties())
+            self.assertIsNotNone(backend.properties(datetime=datetime.today()))
```

Comment:
I suggest giving this check a bit more content, for example:
```
properties_now = backend.properties(datetime=datetime.now())
self.assertIsNotNone(properties_now)
self.assertEqual(properties_now, backend.properties())
properties_old = backend.properties(datetime=datetime.now() - timedelta(days=365))
self.assertIsNotNone(properties_old)
self.assertNotEqual(properties_old, properties_now)
```

Reply:
Not directly related, but when I ran the above suggestion, I got a deprecation warning on 
`target.aquire_alignment`. So I suggest you fix this as well - in `qiskit_ibm_runtime/utils/backend_converter.py`, simply change `aquire_alignment` to `acquire_alignment`.